src/main/java/cz/startnet/utils/pgdiff/PgDiffViews.java

```diff
@@ -72,11 +72,12 @@ public static void createViews(final PrintWriter writer,
      * @param writer           writer the output should be written to
      * @param oldSchema        original schema
      * @param newSchema        new schema
-     * @param searchPathHelper search path helper
+     * @param searchPathHelper search path helper     
```

Comment:
If you are using a formatter, it's missing trailing whitespace

Reply:
I'm not using a different formatter. This formatter error was my fault, when I removed the extra parameter that I put.